db/log_reader.cc

```diff
@@ -36,37 +36,13 @@ Reader::Reader(std::shared_ptr<Logger> info_log,
       eof_offset_(0),
       last_record_offset_(0),
       end_of_buffer_offset_(0),
-      initial_offset_(initial_offset),
       log_number_(log_num),
       recycled_(false) {}
 
 Reader::~Reader() {
   delete[] backing_store_;
 }
 
-bool Reader::SkipToInitialBlock() {
```

Comment:
I just realized that we may still need a `SkipToBlockOffset(uint64_t offset)` function. Since it's not used at present, I can add it when I need it. You don't have to add it now.

Reply:
Oh ok, we can add it back later then :).